### ISSUE TITLE
Port management between instances

### ISSUE THREAD

**Issue by psihius** [factoriommo/factorio-multienv-ctl#9]
Well, I run into this myself with the latest event - I forgot to define specific ports for each server, and had to read the log to see that I derped.
I voiced an idea to put a file into the instance config folder like "port.xxxxx" and read those when creating a new instance, so we can assign a unique port to the instance.

**Comment by maikelwever** [comment]
Would it also be okay to re-use the existing `SERVER_PORT=` parameter in init.conf?

**Comment by psihius** [comment]
Yes, reusing `SERVER_PORT` is best, just need to make sure it does not collide.
Extra level - reuse ports that were used by deleted instances :)

**Comment by maikelwever** [comment]
Using a non-default port does not have any implication on the game: eg: the game can still be found in the server browser?

**Comment by zackman0010** [comment]
Correct, port does not affect the ability to be seen in the server browser

**Comment by psihius** [comment]
Ofcourse, the game is connecting to the server browser and telling it where it can find it, not the other way around. Otherwise you could not run multiple servers on a single IP address :)
